src/components/close-button/close-button.css

```diff
@@ -16,21 +17,29 @@
 
 .close-button.large:hover {
     transform: scale(1.1, 1.1);
+    box-shadow: 0 0 0 4px $ui-black-transparent;
 }
 
 .close-button.large.orange:hover {
     transform: scale(1.1, 1.1);
+    box-shadow: 0px 0px 0px 4px hsla(29, 100%, 54%, 0.2);
 }
 
 .small {
     width: 0.825rem;
     height: 0.825rem;
+    background-color: $motion-primary;
     color: $ui-white;
 }
 
 .large {
     width: 1.75rem;
     height: 1.75rem;
+    box-shadow: 0 0 0 2px $ui-black-transparent;
+}
```

Comment:
Tiny nit, should there be a newline between these?

Reply:
Yes, added!